src/material.cpp

```diff
@@ -185,6 +187,59 @@ Material::Material(pugi::xml_node node)
         } else {
           densities.push_back(-std::stod(get_node_value(node_nuc, "wo")));
         }
+        
+        // cvmt: read and store polynomial represetation of nuclide densities
+        if(check_for_node(node_nuc, "poly_coeffs")){
+          continuous_num_density_ = true;
+          auto type     = get_node_value(node_nuc, "poly_type");
+          auto coeffs   = get_node_array<double>(node_nuc, "poly_coeffs");
+          auto n_coeffs = coeffs.size();
+          struct PolyProperty temp_poly; 
+          //
+          temp_poly.type_     = type;
+          temp_poly.n_coeffs_ = n_coeffs;
```

Comment:
All of this code here and below looks like it should be in a `PolyProperty` constructor that takes a few arguments